pkg/collector/checkhelpers/backoff/backoff.go

```diff
@@ -0,0 +1,167 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package backoff
```

Comment:
Consider using `pkg/util/backoff` instead. If that doesn't work for you, please move your module to `corechecks/cluster/ksm` or `pkg/util`, since this package isn't really specific to checks.